OpenSim/Tools/InverseKinematicsTool.h

```diff
@@ -22,10 +22,10 @@
  * See the License for the specific language governing permissions and        *
  * limitations under the License.                                             *
  * -------------------------------------------------------------------------- */
-
+#include "Simbody.h"
```

Comment:
That is true, I followed suit with similar usage because headers that defines templates are tricky to surgically include but I can look into it further if you think it's worth it.

Reply:
Will `#include <SimTKcommon/internal/ReferencePtr.h>` work?